app/models/form526_submission.rb

```diff
@@ -563,6 +566,12 @@ def submit_flashes
     BGS::FlashUpdater.perform_async(id) if user && Flipper.enabled?(:disability_compensation_flashes, user)
   end
 
+  def poll_form526_pdf
+    # In order to track the status of the 526 PDF upload via Lighthouse,
+    # call poll_form526_pdf, provided we received a valid claim_id from Lighthouse
+    Lighthouse::PollForm526Pdf.perform(id) if submitted_claim_id
```

Comment:
Shouldn't this be `.perform_async(id)` instead?

Reply:
Whoops, a relic from my local testing. Will fix!